Source/AudibleUtilities/ApiExtended.cs

```diff
@@ -99,7 +101,7 @@ private async Task<List<Item>> getItemsAsync(LibraryOptions libraryOptions, bool
 
 			//Scan the library for all added books.
 			//Get relationship asins from episode-type items and write them to episodeChannel where they will be batched and queried.
-			await foreach (var item in Api.GetLibraryItemsPagesAsync(libraryOptions, BatchSize, semaphore))
+			await foreach (var item in GetItems())
```

Comment:
One of us is missing something. This looks like an unconditional throw on ln 88

Reply:
@rmcrackan You just happened to look at an embarrassing time! I'm debugging #1191 and I'm about to throw my hands up. The last thing I want to try is to see how Libation on mac behaves with an ApiErrorException thrown in that spot. Due to some difficulties I'm having getting and IDE running on my mac (due to me not being a mac person), I thought the path of least resistance was to hard-code an exception and let the github runner build it for me.

Reply:
I thought I maybe was being a backseat driver here. I just knew that when the big PR does come, I'm not carefully combing through the hundreds of upcoming changes so I wanted to point this out while I noticed it and save you from a potential d'oh moment. I'll leave you alone :)